templates/ai/data-science.html

```diff
@@ -0,0 +1,495 @@
+{% extends "ai/base_ai.html" %}
+
+{% block title %}Data science on Ubuntu made easy{% endblock %}
+
+{% block meta_description %}
+  Set up ML environments with ease on your AI workstation with an out-of-the-box solution for data science on Ubuntu
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1GdVmQD9EDf_ksN6r1qWTYSwvdynCzAE8wtKaEdtzDo8/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="p-section--shallow">
+          <h1>Data Science Stack on Ubuntu</h1>
+        </div>
+        <div class="p-section--shallow">
+          <p>Set up ML environments with ease on your AI workstation using an out-of-the-box solution for data science.</p>
+        </div>
+        <hr />
+        <a class="p-button--positive"
+           href="https://documentation.ubuntu.com/data-science-stack/en/latest/tutorial/">Try the  Beta</a>
+        <a href="/engage/data-science-tools">Learn more about  data science tools on Ubuntu&nbsp;&rsaquo;</a>
+      </div>
+    </div>
+    <div class="u-fixed-width u-no-padding u-hide--small u-hide--medium">
+      <img src="https://assets.ubuntu.com/v1/7f34ade9-website_suru_25.jpg"
+            alt=""
+            style="aspect-ratio: 5.1775700934579;
+                  width: 100%" />
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr />
```

Comment:
I remember discussing with @lyubomir-popov that we were deprecating the `p-rule` class, but I could be mistaken

Reply:
yes we decided to drop it for simplicity's sake

Reply:
Oh good to know! I'll carry this convention moving forward then. @lyubomir-popov it'd be great if this could be documented on vanilla docs. Couldn't find any mention of it being deprecated on https://vanillaframework.io/docs/whats-new or https://vanillaframework.io/docs/patterns/rule